crates/gosub-bindings/include/properties.h

```diff
@@ -0,0 +1,16 @@
+#ifndef GOSUB_API_PROPERTIES_H
+#define GOSUB_API_PROPERTIES_H
+
+struct position_t {
+  double x;
+  double y;
+};
+
+struct rectangle_t {
+  double top;
+  double left;
+  double right;
+  double bottom;
+};
+
+#endif
```

Comment:
![2023-12-02_12-31](https://github.com/gosub-browser/gosub-engine/assets/760949/d60b7b74-b829-4a7b-9e79-5bb9d28cb459)

Reply:
Looks like the other files have an extra newline except for this one, maybe that's the issue?

Reply:
Not a huge thing.  It's the kind of thing editors usually take care of. But because I'm kind of detail oriented, the red circle in the GitHub view always catches my eye.